shielder/cli/Cargo.toml

```diff
@@ -38,8 +38,11 @@ ark-snark = { version = "^0.3.0", default-features = false }
 ark-std = { version = "^0.3.0", default-features = false }
 
 # todo: change this for versioned `aleph-client` right away after publishing it
-aleph_client = { git = "https://github.com/Cardinal-Cryptography/aleph-node/" }
+aleph_client = { git = "https://github.com/Cardinal-Cryptography/aleph-node.git", rev = "r-10.0" }
```

Comment:
Yup, I introduced this precisely because `ink-wrapper` works with this specific version. Could potentially make this more recent, but this would require some work on `ink-wrapper`.

Reply:
I'm all for keeping aleph-client up-to-date, but `a5199c8 or newer.` is not a version :P Can we please, please, with a cherry on top, have semver for aleph-client finally?

Reply:
`2.15.1` better? :D

Reply:
I'll try get things moving

Reply:
:pray:

Reply:
@pmikolajczyk41 Can I merge this as is? I take it it will take some time before we have `aleph-client` available on `crates.io`.